sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/dynamicconfig/nacos/NacosClient.java

```diff
@@ -325,10 +325,12 @@ private String doRequest(String url) throws IOException {
             HttpGet httpGet = new HttpGet(url);
             httpGet.setConfig(requestConfig);
             try (CloseableHttpResponse response = httpClient.execute(httpGet)) {
-                HttpEntity entity = response.getEntity();
-                result = EntityUtils.toString(entity);
+                if (response.getStatusLine().getStatusCode() == SUCCESS_CODE) {
+                    return EntityUtils.toString(response.getEntity());
+                }
+                LOGGER.error("Http Request error, the message is: {}", EntityUtils.toString(response.getEntity()));
```

Comment:
describe it more clearly, it is only used by getGroupKeys

Reply:
done